scripts/xe-backup-metadata

```diff
@@ -74,7 +74,7 @@ function validate_vdi_uuid {
 function test_sr {
   sr_uuid_found=$(${XE} sr-list uuid="$1" --minimal)
   if [ "${sr_uuid_found}" != "$1" ]; then
-     echo Invalid SR UUID specified: $1
+     echo "Invalid SR UUID specified: $1"
```

Comment:
missing hash of original commit in the commit message: f88fa12582e0e729b36d048b1bf14268ee9746c0